awx/main/management/commands/host_metric.py

```diff
@@ -0,0 +1,66 @@
+from django.core.management.base import BaseCommand
+from dateutil import parser
+from datetime import timezone
+from awx.main.models.inventory import HostMetric
+import json
+
+
+class Command(BaseCommand):
+
+    help = 'This is for offline licensing usage'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--since', dest='since', action='store', help='Start date for data')
```

Comment:
Now that we're on Python3.8, you could probably do something like this instead:

`parser.add_argument('--since', type=datetime.date.fromisoformat)`

Reply:
```[yossarian] (fedora:●●) ~ python test.py
{'since': None}
[yossarian] (fedora:●●) ~ python test.py --since 2021-01-01
{'since': datetime.date(2021, 1, 1)}
[yossarian] (fedora:●●) ~ python test.py --since 2021-01-dog
usage: test.py [-h] [--since SINCE]
test.py: error: argument --since: invalid fromisoformat value: '2021-01-dog'```

Reply:
I have made the changes.